src/components/PivotTable/Droppable/Droppable.test.tsx

```diff
@@ -0,0 +1,184 @@
+      keys={keys}
+      handleKeyUpdate={() => {}}
+      handleFilterUpdate={() => {}}
+      {...props}
+    />
+    <Droppable<Fruit>
+      name={'droppable-2'}
+      keyState={[]}
+      filterState={new Map<keyof Fruit, Set<string>>()}
+      hasFilter={false}
+      type={'fruit-table'}
+      keyMapping={keyMapping}
+      keys={keys}
+      handleKeyUpdate={() => {}}
+      handleFilterUpdate={() => {}}
+      {...props}
+    />
+  </DndProvider>
+)
+
+const createFilterComponent = (props: Partial<DroppableProps<Fruit>> = {}) => (
+  <DndProvider backend={HTML5Backend}>
+    <Droppable<Fruit>
+      name={'droppable-1'}
+      keyState={['name']}
+      filterState={new Map<keyof Fruit, Set<string>>()}
+      hasFilter={true}
+      type={'fruit-table'}
+      keyMapping={keyMapping}
+      keys={keys}
+      handleKeyUpdate={() => {}}
+      handleFilterUpdate={() => {}}
+      {...props}
+    />
+    <Droppable<Fruit>
+      name={'droppable-2'}
+      keyState={[]}
+      filterState={new Map<keyof Fruit, Set<string>>()}
+      hasFilter={true}
+      type={'fruit-table'}
+      keyMapping={keyMapping}
+      keys={keys}
+      handleKeyUpdate={() => {}}
+      handleFilterUpdate={() => {}}
+      {...props}
+    />
+  </DndProvider>
+)
+
+describe('Draggable', () => {
+  describe('render', () => {
+    it('should render without filter options', () => {
+      const { container } = render(createDefaultComponent())
+      expect(container).toMatchSnapshot()
+    })
+    it('should render with filter options', () => {
+      const { container } = render(createFilterComponent())
+      expect(container).toMatchSnapshot()
+    })
+  })
+  describe('Drag and drop', () => {
+    it('should call onDragEnd when the drag event ends in the same droppable', () => {
+      const handleKeyUpdate = jest.fn()
+
+      const { container } = render(createDefaultComponent({ handleKeyUpdate: handleKeyUpdate }))
+
+      const droppable = container.firstChild
+      const dragabble = droppable.firstChild.firstChild
+
+      fireEvent.dragStart(dragabble)
+      fireEvent.dragEnter(droppable)
+      fireEvent.dragOver(droppable)
+      fireEvent.drop(droppable)
+      fireEvent.dragEnd(dragabble)
+
+      expect(handleKeyUpdate).toHaveBeenCalled()
+    })
+
+    it('should call onDragEnd when the drag event ends for droppable with filter in the same droppable', () => {
+      const handleKeyUpdate = jest.fn()
+
+      const { container } = render(createFilterComponent({ handleKeyUpdate: handleKeyUpdate }))
+
+      const droppable = container.firstChild
+      const dragabble = droppable.firstChild.firstChild
+
+      fireEvent.dragStart(dragabble)
+      fireEvent.dragEnter(droppable)
+      fireEvent.dragOver(droppable)
+      fireEvent.drop(droppable)
+      fireEvent.dragEnd(dragabble)
+
+      expect(handleKeyUpdate).toHaveBeenCalled()
+    })
+
+    it('should call onDragEnd when the drag event ends for droppable with filter', () => {
```

Comment:
Acho que esse título ficaria mais descritivo desta forma: `(...) drag event ends in another dropable with filter`